### ISSUE TITLE
Error Integrating complex integrand

### ISSUE THREAD

**Issue by KingHarley** [nschloe/quadpy#245]
I obtain a **TypeError** when trying to integrate a complex integrand.

Below is my simple script written in Visual Studio's interactive window. To install quadpy I used Visual Studio's in built ``pip install quadpy``

Note that integration of a non-complex integrand works, seen as the result for ``test``. However, the complex integrand throws the error (``test2``).

```py
SpinWaveSimulation interactive window [PTVS 16.4.19310.1-16.0]
Type $help for a list of commands.
>>> import numpy
>>> import quadpy
>>> test = quadpy.quad(lambda x : x ** 3, 0, 1)
>>> test
(array(0.25), array(9.99819478e-20))
>>> test2 = quadpy.quad(lambda x : x + x * 1j, 0, 1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\22159666\AppData\Local\Continuum\anaconda3\lib\site-packages\quadpy\_scipy_compat.py", line 44, in quad
    g, [a, b], eps_abs=epsabs, eps_rel=epsrel, max_num_subintervals=limit
  File "C:\Users\22159666\AppData\Local\Continuum\anaconda3\lib\site-packages\quadpy\line_segment\_tools.py", line 69, in integrate_adaptive
    total_val[..., is_good] += val[..., is_good]
TypeError: Cannot cast ufunc add output from dtype('complex128') to dtype('float64') with casting rule 'same_kind'
>>> ```

**Comment by nschloe** [comment]
Thanks, I can reproduce the error. Looking into it right now...

**Comment by nschloe** [comment]
Alright, the fix is in 0.14.5. Thanks again for the report!

**Comment by KingHarley** [comment]
Thank you very much Nico!
